test_manager/features/tests/transport_rpc/register_and_send.feature

```diff
@@ -25,58 +25,42 @@
 Feature: Testing Publish and Subscribe Functionality
 
   Scenario Outline: To test the registerlistener and send apis
-    Given "<uE1>" creates data for "registerlistener"
-    And sets "entity.name" to "body.access"
-    And sets "entity.id" to "12345"
-    And sets "entity.version_major" to "1"
-    And sets "resource.name" to "door"
-    And sets "resource.id" to "12345"
-    And sets "resource.instance" to "front_left"
-    And sets "resource.message" to "Door"
+    Given "uE1" creates data for "registerlistener"
+    And sets "authority_name" to "me_authority"
+    And sets "ue_id" to "23456"
+    And sets "ue_version_major" to "1"
+    And sets "resource_id" to "32769"
 
     When sends "registerlistener" request
     Then the status received with "code" is "OK"
 
-    When "<uE2>" creates data for "send"
-    And sets "attributes.id.msb" to "112128268635242497"
-    And sets "attributes.id.lsb" to "11155833020022798372"
-    And sets "attributes.source.entity.name" to "body.access"
-    And sets "attributes.source.entity.id" to "12345"
-    And sets "attributes.source.entity.version_major" to "1"
-    And sets "attributes.source.resource.name" to "door"
-    And sets "attributes.source.resource.id" to "12345"
-    And sets "attributes.source.resource.instance" to "front_left"
-    And sets "attributes.source.resource.message" to "Door"
+    When "uE2" creates data for "send"
+    And sets "attributes.id.msb" to "112808788591603906"
+    And sets "attributes.id.lsb" to "11713802770567977244"
+    And sets "attributes.source.authority_name" to "me_authority"
+    And sets "attributes.source.ue_id" to "23456"
+    And sets "attributes.source.ue_version_major" to "1"
+    And sets "attributes.source.resource_id" to "32769"
     And sets "attributes.priority" to "UPRIORITY_CS1"
     And sets "attributes.type" to "UMESSAGE_TYPE_PUBLISH"
-    And sets "payload.format" to "UPAYLOAD_FORMAT_PROTOBUF_WRAPPED_IN_ANY"
-    And sets "payload.value" to b".type.googleapis.com/google.protobuf.Int32Value\x12\x02\x08\x03"
+    And sets "attributes.commstatus" to "OK"
```

Comment:
attributes.commstatus is not allowed in send message

```suggestion

```

Reply:
got it, thanks, removed